elastic/kibana
# Add ".reindexed-v8-" prefix to the valid prefixes list

As part of v9.0 readiness, we reindex the indices in v.8 but still has data from v.7x.
As a result of the process, the reindexed indices get `.reindexed-v8-` prefix.
This PR add that prefix to the valid prefixes list.


# To verify:

Run Kibana and ES in 7.17 (use `-E path.data=your-data-path` to keep the data in your local)
Create some rules that generate alerts (Observability rules to have AAD)
Let them run for a while.
Stop Kibana and ES, switch to 8.x branch and run ES and Kibana again
Open the Upgrade Assistant.
It should show the `.internal.alerts-*` indices 
Click on them and start reindexing on the opened flyout.
Check that `.reindexed-v8-internal.alerts-*` index has been created
Let you rules run for a while again.
Your alerts should be updated and there shouldn't be any error on your terminal.
